recipes/xspec-models-lite/meta.yaml

```
@@ -0,0 +1,76 @@
+{% set name = "xspec-models-lite" %}
+{% set version = "6.22" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+
+  url: https://heasarc.gsfc.nasa.gov/FTP/software/lheasoft/lheasoft6.22.1/xspec-modelsonly-v6.22.1.tar.gz
```

Using jinja2 variables can avoid mistakes when updating:
```suggestion
  url: https://heasarc.gsfc.nasa.gov/FTP/software/lheasoft/lheasoft{{ version }}/xspec-modelsonly-v{{ version }}.tar.gz
```